### PR TITLE
[docs] port hello_blockchain to being purely cli tutorial

### PR DESCRIPTION
**Pull request by davidiw**
this is a pretty rough doc, but my goal is to just power through and hopefully massage a bit...

considering that we're going to be disabling the module publishing transaction very soon...

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3384)

### CODE REVIEW

File: developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md [hunk 1]
```diff
@@ -18,6 +17,21 @@ This document describes how to install the `aptos` CLI tool. See [Use Aptos CLI]
 6. Type `~/bin/aptos help` to read help instructions.
 7. Add `~/bin` to your path in your appropriate `.bashrc` or `.zshrc` for future use.
 
+## Install from Git
+
+Start by cloning the `aptos-core` GitHub repo from [GitHub](https://github.com/aptos-labs/aptos-core).
```
Comment (gregnazario) [inline developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md:22]: I guess cause you need this stuff for cargo anyways

File: developer-docs-site/docs/tutorials/first-move-module-cli.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+
+No key given, generating key...
+Account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a doesn't exist, creating it and funding it with 10000 coins
+Aptos is now set up for account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a!  Run `aptos help` for more information about commands
+{
+  "Result": "Success"
+}
+```
+2. Now fund this account: `aptos account fund-with-faucet --account a345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a`
```
Comment (gregnazario) [inline developer-docs-site/docs/tutorials/first-move-module-cli.md:42]: The previous step should probably find enough for the tutorial?  It'll even fund an account that already has coins

**Comment by jjleng** [comment]
If we disable module publishing txn, we should allow users to publish packages through code (if the package metadata is available on disks)

**Review by jjleng — APPROVED** [review]

File: developer-docs-site/docs/tutorials/first-move-module-cli.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+```
+
+## Compile and test the module
+
+There are many example Move modules in the [aptos-core/aptos-move/move-examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples) directory.
+
+Load a terminal and change directories into the `hello_blockchain` directory: `cd aptos-core/aptos-move/move-examples`.
+
+To build the module run: `aptos move build --named-addresses hello_blockchain=0xa345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a`.
+To test the module run: `aptos move test --named-addresses hello_blockchain=0xa345dbfb0c94416589721360f207dcc92ecfe4f06d8ddc1c286f569d59721e5a`.
```
Comment (gregnazario) [inline developer-docs-site/docs/tutorials/first-move-module-cli.md:56]: Should you use the alias from the init step?  E.g --named-addresses hello_blockchain=default

Comment (davidiw) [reply developer-docs-site/docs/tutorials/first-move-module-cli.md:56]: I'll land and leave it to you folks to adjust according to what you know.

**Comment by davidiw** [comment]
There's a lot to the CLI, I don't know. I just wanted to put the story board there. I think @rajkaramchedu will do a great job improving the writing and the platform folks will do a great job making sure it is as accurate and concise as possible.

**Pull request merged**